src/main/kotlin/com/exactpro/th2/common/value/ValueFilterUtils.kt

```diff
@@ -56,6 +57,8 @@ fun Any.toValueFilter(): ValueFilter = when (this) {
     is ListValueFilter.Builder -> toValueFilter()
     is Iterator<*> -> toValueFilter()
     is Iterable<*> -> toValueFilter()
+    is Sequence<*> -> iterator().toValueFilter()
+    is Stream<*> -> iterator().toValueFilter()
```

Comment:
Maybe use a base approach for Stream and Sequence. I mean `.map(toValue).forEach(valueList::add)`

Reply:
To do this we'll have to create `ListValue` first. So it's just better to reuse existing function which does exactly that